src/components/templates/news-and-events/newsAndEvents.html

```diff
@@ -0,0 +1,89 @@
+<section class="newsAndEvents container">
```

Comment:
```suggestion
<section class="news-and-events container">
```

クラス名は基本的にケバブケースでお願いします。